index.d.ts

```diff
@@ -283,7 +283,7 @@ declare namespace algoliasearchHelper {
      * @deprecated since version 2.4.0, see {@link AlgoliaSearchHelper#addFacetExclusion}
      */
     addExclude: AlgoliaSearchHelper['addFacetExclusion'];
-    addTag(facet: string, value: string): this;
+    addTag(facet: string): this;
```

Comment:
Nitpicky suggestion:

```suggestion
    addTag(value: string): this;
```

or, if we follow https://github.com/algolia/algoliasearch-helper-js/blob/develop/src/algoliasearch.helper.js#L581:

```suggestion
    addTag(tag: string): this;
```

Reply:
good call, thanks, should have removed the first, not the second part :D